pkg/config/config_template.yaml

```diff
@@ -2834,6 +2834,42 @@ api_key:
   #   - '<COMMUNITY_1>'
   #   - '<COMMUNITY_2>'
 
+  ## @param users - list of custom objects - optional
+  ## List of SNMPv3 users that can be used to listen for traps.
+  ## NOTE: Currently the Datadog Agent only supports having a
+  ## single user in this list.
+  #
+  # users:
+
+    ## @param username - string - optional
+    ## The username used by devices when sending TRAPs to the Agent.
+    #
+    # username: <USERNAME>
```

Comment:
Should be a list of users to match implementation ?

Reply:
yes